projects/cmake/build.sh

```diff
@@ -188,14 +182,11 @@ mv GitVersion.cpp ../../src/revlanguage/utils/
 
 
 ######### Generate help database
-if [ "$help2yml" = "true" ]
-then
-    (
-        cd ../../src
-        echo "Generating help database"
-        perl ../help/md2help.pl ../help/md/*.md > core/help/RbHelpDatabase.cpp
-    )
-fi
+(
+cd ../../src
+echo "Generating help database"
```

Comment:
Here can we just call the generate_help.sh script?

Reply:
Sure! Will change